### ISSUE TITLE
Re-vision Twixtor freeze Natron 2.1.0 shapshot with GPU ON

### ISSUE THREAD

**Issue by adallafontana** [MrKepzie/Natron#1339]
Natron-201606091348-a9589bee606d7950f302553f95b59a00a0cea992-Linux-x86_64bit

OS Ubuntu 16.04   Nvidia 361.42  Quadro 4000 - GF 780M

Tested on 2 machines, GPU render ON:

One or multiple render seems to not be responding anymore after numerous attempt made by Natron to abort them for the last 5.00 seconds.
This is likely due to a render taking too long in a plug-in.

List of stalled render(s):
 - Global Thread (Pooled) (0x7feef000da78) stalled in:
    Node: Twixtor1
    Plugin: com.revisionfx.Twixtor
    Action: kOfxImageEffectActionRender
Would you like to kill these renders?
WARNING: Killing them may not work or may leave Natron in a bad state. The application may crash or freeze as a consequence of this. It is advised to restart Natron instead.

With CPU render Twixstor works right. I don't understant if GPU render is supported or not in Natron yet.

**Comment by MrKepzie** [comment]
Yes we added GPU support for 2.1. This will be considered a beta feature throughout 2.1, but will be fully stable for 2.2. The first effect that we did that supports GPU is called "ShaderToy" and is much like SeExpr, except that the script you input is interpreted as a GLSL shader and executed on the GPU (so it's roughly 100 times faster than SeExpr).

If all goes well, for 2.2 we gonna be migrating a lot of effects to GPU
